### PR TITLE
remove creation of reactor-config secret, remove deprecated values fr…

### PR DESCRIPTION
**Pull request by rcerven**
…om client-config secret

Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by twaugh** [comment]
Why are we removing the reactor-config (YAML) secret? I thought it would be the client-config (osbs.conf) secret we'd be removing. Have I misunderstood?

**Comment by rcerven** [comment]
we are now using reactor-config-map (configmap) and not reactor-config secret

and we aren't using client-config because we use user params

so I am removing both of them

File: README.md [hunk 1]
```diff
@@ -11,8 +11,6 @@ If user is cluster admin (`osbs_is_admin`), the following is also performed:
 - Sets up rolebindings for specified users, groups and service accounts
 
 For orchestrator namespaces (`osbs_orchestrator`):
-- reactor-config-secret is generated and stored in `osbs_generated_config_path`
-  use osbs-secret to import it
 - client-config-secret is generated and stored in `osbs_generated_config_path`
   use osbs-secret to import it
```
Comment (twaugh) [inline README.md]: Should `- client-config-secret is generated ...` also be removed?

Comment (rcerven) [reply README.md]: yes thanks, removed

**Pull request merged**